src/components/experiment-wrapper/index.js

```diff
@@ -12,22 +12,39 @@ const MAX_NUMBER_COMPARISONS = 2; // 0-based, so three
 
 const ExperimentWrapper = ({ theme }) => {
   const [isSidebarVisible, setIsSidebarVisible] = useState(true);
+  const [enableShowChanges, setEnableShowChanges] = useState(true);
   const [disableRunSelection, setDisableRunSelection] = useState(false);
   const [enableComparisonView, setEnableComparisonView] = useState(false);
   const [selectedRuns, setSelectedRuns] = useState([]);
+  const [pinnedRun, setPinnedRun] = useState();
 
   const { data, loading } = useApolloQuery(GET_RUNS);
 
+  // runs needs to be sorted by time to ensure runIDs get sent to
+  // graphql endpoint in correct order
+  const sortRunByTime = (runs) => {
+    const runsWithTimestamps = runs.map((run) => ({
+      id: run,
+      dateObj: new Date(run.replace('.', ':').replace('.', ':')),
```

Comment:
You could probably put this part `run.replace('.', ':').replace('.', ':')` in our `src/utils/date-utils.js` file, as it's also used within a function there.

Reply:
great suggestion - I had incorporated this with some further refactor by extracting the `sortRunByTime` function into `date-utils` as well :)